packages/orm/src/createOrUpdatePartial.ts

```diff
@@ -122,37 +122,41 @@ export async function createOrUpdatePartial<T extends Entity>(
       const allowDelete = !field.otherMetadata().fields["delete"];
       const allowRemove = !field.otherMetadata().fields["remove"];
       const allowOp = !field.otherMetadata().fields["op"];
-      collectionsToLoad.push(field.fieldName);
+      relationsToLoad.push(field.fieldName);
 
-      const entities = !value
-        ? []
-        : (value as Array<any>).map(async (value) => {
-            if (!value || isEntity(value)) {
+      const entities = await Promise.all(
+        toArray(value).map(async (value: any) => {
+          if (!value || isEntity(value)) {
```

Comment:
This diff looks large, but really just removed the `!value ? [] : ...cast...` with `toArray` and everything else here is indentation change.